engine.py

```diff
@@ -357,45 +372,19 @@ def update_shotgun_menu(self):
     #                         Python commands are always executed in the main 3ds Max thread.
     #                         You should not attempt to spawn separate threads in your scripts
     #                         (for example, by using the Python threading module).
-    def log_debug(self, msg):
+    def _emit_log_message(self, handler, record):
         """
-        Debug logging.
-        :param msg: The message string to log
+        Emits a log message.
         """
-        if self.get_setting("debug_logging", False):
-            self.async_execute_in_main_thread(
-                self._print_output, "Shotgun Debug: %s" % msg
-            )
-
-    def log_info(self, msg):
-        """
-        Info logging.
-        :param msg: The message string to log
-        """
-        self.async_execute_in_main_thread(self._print_output, "Shotgun Info: %s" % msg)
-
-    def log_warning(self, msg):
-        """
-        Warning logging.
-        :param msg: The message string to log
-        """
-        self.async_execute_in_main_thread(
-            self._print_output, "Shotgun Warning: %s" % msg
-        )
-
-    def log_error(self, msg):
-        """
-        Error logging.
-        :param msg: The message string to log
-        """
-        self.async_execute_in_main_thread(self._print_output, "Shotgun Error: %s" % msg)
+        msg_str = handler.format(record)
+        self.async_execute_in_main_thread(self._print_output, msg_str)
```

Comment:
The new style logging takes care of formatting the messages, so we use the handler's formatter now.